homeassistant/components/http.py

```diff
@@ -202,6 +202,11 @@ def register_path(self, method, url, callback, require_auth=True):
         """ Regitsters a path wit the server. """
         self.paths.append((method, url, callback, require_auth))
 
+    def log_message(self, fmt, *args):
```

Comment:
log_message is the function that the BaseHTTP objects have built in to write their output to the screen. This forces the HTTP Server to use the same logging as has been configured for the rest of the application. If we don't, then HTTP Server messages will always be written to the screen. Basically, this implements the functionality that the built-in HTTP Server objects should have built in, but don't.

As an aside, this is also the reason that the HTTP Server messages are always formatted differently form all the other messages.